### ISSUE TITLE
[Extreme Reactors] some Fluidizer dont have recipes in EMI

### ISSUE THREAD

**Issue by criscky** [EnigmaticaModpacks/Enigmatica10#228]
### Modpack Version

1.13.0

### Describe your issue.

![image](https://github.com/user-attachments/assets/9c088011-5ffb-4ef0-abd1-02f81ef39d03)
![image](https://github.com/user-attachments/assets/873e0ed9-dec9-4c0d-a9d8-4939c39c52dd)
![image](https://github.com/user-attachments/assets/d21c0232-6ec2-4192-bf97-7034ac6c41f4)

The blocks above dont have recipes

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

### Discord Username

criscky

**Comment by MuteTiefling** [comment]
Yeah I don't think these are fully implemented by the mod. Someone correct me if I'm wrong though.

**Comment by criscky** [comment]
yes. it shows in JEI without EMI, and that brought up another issue
it shows those recipes needing yellorium not uranium, because in the json recipe they put the tag for yellorium, not uranium

**Comment by theboo** [comment]
Oh, that might be why they aren't showing... We have yellorium disabled and EMI doesn't show recipes with disabled items

**Comment by criscky** [comment]
yes, as a note the recipe is tag based, but tagged as "c:ingots/yellorium"

**Comment by theboo** [comment]
Ope I was too slow with my edit lol

**Comment by theboo** [comment]
Alternatively, we could just add the uranium tag to the yellorium tag, this way it would be fixed in any other recipes this might exist in that we might be missing

**Comment by criscky** [comment]
it's not the oposite, like add the yellorium tag to the uranium tag?
because i'm pretty sure yellorium already have uranium tag

**Comment by theboo** [comment]
Yellorium is in the uranium tag yes, but I'm saying if we added the uranium tag(not the item, the tag) to the yellorium tag,  then any of these recipes that use the yellorium tag would work with anything in the uranium tag because now the yellorium tag would include anything inside the uranium tag.

Adding yellorium tag to the uranium tag would do nothing because these recipes use the yellorium tag, not the uranium..

As an aside, I have now typed the word "tag" so much it has lost all meaning lol

**Comment by criscky** [comment]
yes, i understand, but also i find it funny how many times the word tag was used
